Update to 0.54.1

synapticarbors opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

synapticarbors commented:
@conda-forge-admin, please rerender

synapticarbors commented:
@conda-forge/fastapi -- This feedstock has gotten several versions out of date. I've updated the recipe and added myself as a co-maintainer. I removed the pyproject-patch and just made `flit` a `host` dependency, which seemed to work. I'm not sure about the history of removing the pyproject file, but it doesn't appear to be necessary any more.

Please have a look at this PR when you get a chance. Thanks

dhirschfeld commented:
I'd been meaning to look into updating this feedstock and removing the `pyproject` hack but that was just one more thing I hadn't gotten around to so thanks @synapticarbors - will be good having you on-board!
